docker-compose.yml

```
@@ -0,0 +1,37 @@
+services:
+  meet-teams-bot:
+    build: .
+    image: meet-teams-bot
+    container_name: meet-teams-bot
+    stdin_open: true
+    tty: true
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)